x-pack/platform/test/saved_object_api_integration/low-level/apis/find.ts

```diff
@@ -0,0 +1,394 @@
+            `${getUrlPrefix(defaultNamespace)}/api/saved_objects/_find?${query}`
+          );
+          expect(response.status).to.eql(400);
+          expect(response.body.message).to.match(
+            /Sort field "sortTestingType.textWithKeyword" is of type "text" which is not sortable. If the field has a sortable subfield e.g "keyword" subfield, use "field.keyword" for sorting./
+          );
+        });
+
+        it('throws error if sorting field is missing in one of the types', async () => {
+          const query = queryString.stringify({
+            type: sortTestTypes,
+            sort_field: sortTestFields.missing,
+          });
+          const response = await supertest.get(
+            `${getUrlPrefix(defaultNamespace)}/api/saved_objects/_find?${query}`
+          );
+          expect(response.status).to.eql(400);
+          expect(response.body.message).to.match(
+            /Sort field "notPresent" is not available for all specified types. Each type must have the field defined either at the type level or root level./
+          );
+        });
+
+        it('sorts by _id and _score', async () => {
+          const idQuery = queryString.stringify({
+            type: sortTestTypes,
+            sort_field: sortTestFields.id,
+          });
+          const idResponse = await supertest.get(
+            `${getUrlPrefix(defaultNamespace)}/api/saved_objects/_find?${idQuery}`
+          );
+          expect(idResponse.status).to.eql(400);
+          expect(idResponse.body.message).to.match(
+            /Fielddata access on the _id field is disallowed/
+          );
+
+          const scoreQuery = queryString.stringify({
+            type: sortTestTypes,
+            sort_field: sortTestFields.score,
+          });
+          const scoreResponse = await supertest.get(
+            `${getUrlPrefix(defaultNamespace)}/api/saved_objects/_find?${scoreQuery}`
+          );
+          expect(scoreResponse.status).to.eql(200, JSON.stringify(scoreResponse.body));
+        });
+
+        it('sorts by keyword field for multiple types when some documents are missing the field', async () => {
+          // This assumes that at least one document in sortTestingType or sortTestingType2 is missing 'titleKeyword'
+          // If not, you may need to adjust the fixture data to include such a document.
+          const query = queryString.stringify({
+            type: sortTestTypes,
+            sort_field: sortTestFields.missingData,
+          });
+          const response = await supertest.get(
+            `${getUrlPrefix(defaultNamespace)}/api/saved_objects/_find?${query}`
+          );
+          expect(response.status).to.eql(200, JSON.stringify(response.body));
+          // Documents missing the sort field should appear last in the sort order
+          const titles = response.body.saved_objects.map((o: any) => o.attributes.titleKeyword);
+          // Check that undefined/null values (missing field) are at the end
```

Comment:
What is the fallback sort value for these docs?

Reply:
for `type: keyword` we emit an empty string from the runtime mapping